io/disk/fiotest.py

```diff
@@ -87,6 +90,8 @@ def setUp(self):
                 else:
                     pkg_list.extend(['ndctl', 'daxctl', 'numactl-devel',
                                      'ndctl-devel', 'daxctl-devel'])
+        if raid_needed:
+            pkg_list.append('mdam')
```

Comment:
@narasimhan-v here package  spelling is wrong.

Reply:
thanks, pushed to fix it.